packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -28,8 +28,8 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: ansible
 %else
 Requires: (ansible or ansible-core)
```

Comment:
Does it still require ansible directory or is it always invoking ansible-runner? It looks to me like the latter and we can simply rely on ansible-runner to pull in ansible itself.

Reply:
It will always invoke runner, but the runner pkg has no dependency on Ansible…

Reply:
Is that something we should fix?

Reply:
It's complicated. *Technically* runner doesn't need a packaged Ansible (it can utilize a container instead, for example). However, for our usecases, it does need it, as we don't use any of the container functionality.